docs/guides/quickstarts/react.mdx

```diff
@@ -1,149 +1,308 @@
 ---
-title: Setup Nhost with React
+title: Get up and running with Nhost and React
 sidebarTitle: React
-description: Get up and running with Nhost and React
+description: In this quickstart guide, we'll demonstrate how to build a simple To-Do feature using Nhost and React
 icon: react
 ---
 
+<Card>
```

Comment:
not sure about this card, it gives importance to something that may or may not be relevant to newcomers

Reply:
👌 will remove